airflow/providers/datadog/hooks/datadog.py

```diff
@@ -90,9 +92,9 @@ def send_metric(self, metric_name, datapoint, tags=None, type_=None, interval=No
         return response
 
     def query_metric(self,
-                     query,
-                     from_seconds_ago,
-                     to_seconds_ago):
+                     query: str,
+                     from_seconds_ago: int,
+                     to_seconds_ago: int) - > Dict[str, Any]:
```

Comment:
This function returns a `response` -- would this indeed be a `Dict` type or a custom object?